cli/clilog/clilog.go

```diff
@@ -112,13 +112,17 @@ func (b *Builder) Build(inv *serpent.Invocation) (log slog.Logger, closeLog func
 			sinks = append(sinks, sinkFn(inv.Stderr))
 
 		default:
-			fi, err := os.OpenFile(loc, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0o644)
-			if err != nil {
-				return xerrors.Errorf("open log file %q: %w", loc, err)
+			logWriter := &lumberjack.Logger{
+				Filename: loc,
+				MaxSize:  5, // MB
+				// Without this, rotated logs will never be deleted.
+				MaxBackups: 1,
 			}
-			closers = append(closers, fi.Close)
-			sinks = append(sinks, sinkFn(fi))
+			closers = append(closers, logWriter.Close)
```

Comment:
Does this give us the equivalent behaviour of `lumberjackWriteCloseFixer` in `cli/agent.go`?

Reply:
I think we should probably re-use that same fix here, although the main reason for that seems to have been to reduce CI flakes due to racy writes after close. We probably won't be logging to a file in CI in the general case.